src/OSPSuite.Core/Services/ExplicitFormulaParser.cs

```diff
@@ -46,13 +46,13 @@ public double Compute(double[] variableValues, double[] parameterValues)
             _parserFunc.SetParameterValues(parameterValues);
             return _parserFunc.CalcExpression(variableValues);
          }
-         catch (FuncParserException)
+         catch (OSPSuiteException ex)
          {
-            throw;
+            throw new FuncParserException(ex.Message,ex);
```

Comment:
ok, keeping FuncParserException for now. But the FuncParser itself throws OSPSuiteException in case of (handled) errors - so it must be catched explicitly.